db/model/Chart.ts

```diff
@@ -379,3 +382,54 @@ export const getRelatedChartsForVariable = async (
                 ORDER BY title ASC
             `)
 }
+
+export const getChartEmbedUrlsInPublishedWordpressPosts = async (): Promise<
+    string[]
+> => {
+    const chartSlugQueryString: { target: string; queryString: string }[] = (
+        await db.knexInstance().raw(
+            `
+            SELECT
+                pl.target,
+                pl.queryString
+            FROM
+                posts_links pl
+                JOIN posts p ON p.id = pl.sourceId
+            WHERE
+                pl.linkType = "grapher"
+                AND pl.componentType = "src"
+                AND p.status = "publish"
+                AND p.type != 'wp_block'
+                AND p.slug NOT IN (
+                    -- We want to exclude the slugs of published gdocs, since they override the Wordpress posts
+                    -- published under the same slugs.
+                    SELECT
+                        slug from posts_gdocs pg
+                    WHERE
+                        pg.slug = p.slug
+                        AND pg.content ->> '$.type' <> 'fragment'
+                        AND pg.published = 1
+                )
+        -- Commenting this out since we currently don't do anything with the baked embeds in gdocs posts
+        -- see https://github.com/owid/owid-grapher/issues/2992#issuecomment-1934690219
+        -- Rename to getChartEmbedUrlsInPublishedPosts if we decide to use this
```

Comment:
Can you add a comment in that issue so that, when someone will pick it up, they will find this code quickly and be able to enable baking the thumbnails?